firefox-ios/Client/Protocols/URLSession/URLSessionProtocol.swift

```diff
@@ -21,6 +23,13 @@ protocol URLSessionProtocol: Sendable {
     ) -> URLSessionDataTaskProtocol
 }
 
+/// Default implementation for bytes(for:) (for backward compatibility)
+extension URLSessionProtocol {
```

Comment:
@cyndichin not sure about this one. Mainly added it because `WallpaperURLSessionMock` was complaining about this method not being implemented. I think this is fine since consumers should only use this if it's intentional. What do you think ?